base/math.jl

```diff
@@ -74,15 +74,15 @@ macro evalpoly(z, p...)
         ai = symbol("a", i)
         push!(as, :($ai = $a))
         a = :(muladd(r, $ai, $b))
-        b = :(muladd(-s, $ai, $(esc(p[i]))))
+        b = :($(esc(p[i])) - s * $ai) # don't have efficient multiply-subtract
     end
     ai = :a0
     push!(as, :($ai = $a))
     C = Expr(:block,
              :(x = real(tt)),
              :(y = imag(tt)),
              :(r = x + x),
-             :(s = x*x + y*y),
+             :(s = muladd(x, x, y*y)),
```

Comment:
This breaks the symmetry between `x` and `y`. I assume `@evalpoly` is only used in circumstances where something like `@fastmath` would make sense?